<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1151</width>
    <height>938</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New Genome</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">font: 11pt &quot;Sans Serif&quot;;</string>
   </property>
   <layout class="QGridLayout" name="gridLayout_9">
    <item row="0" column="3">
     <spacer name="horizontalSpacer_4">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
    <item row="0" column="1">
     <layout class="QGridLayout" name="gridLayout_6">
      <item row="8" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QPushButton" name="contButton">
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>746</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="viewStatButton">
          <property name="text">
           <string>View Statistics</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="4" column="0">
       <widget class="QGroupBox" name="Step2">
        <property name="maximumSize">
         <size>
          <width>11111</width>
          <height>153</height>
         </size>
        </property>
        <property name="title">
         <string>Step 2: Select File and Add to Queue</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_3">
         <item row="2" column="3">
          <widget class="QPushButton" name="resetButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Reset Form</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLabel" name="label_7">
           <property name="maximumSize">
            <size>
             <width>325</width>
             <height>23</height>
            </size>
           </property>
           <property name="text">
            <string>Selected File:</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1" rowspan="2">
          <widget class="QScrollArea" name="scrollArea">
           <property name="maximumSize">
            <size>
             <width>325</width>
             <height>79</height>
            </size>
           </property>
           <property name="widgetResizable">
            <bool>true</bool>
           </property>
           <widget class="QWidget" name="scrollAreaWidgetContents">
            <property name="geometry">
             <rect>
              <x>0</x>
              <y>0</y>
              <width>323</width>
              <height>77</height>
             </rect>
            </property>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="s_file">
               <property name="font">
                <font>
                 <family>Sans Serif</family>
                 <pointsize>11</pointsize>
                 <weight>50</weight>
                 <italic>false</italic>
                 <bold>false</bold>
                </font>
               </property>
               <property name="text">
                <string>Name Of File</string>
               </property>
               <property name="alignment">
                <set>Qt::AlignCenter</set>
               </property>
              </widget>
             </item>
            </layout>
           </widget>
          </widget>
         </item>
         <item row="0" column="0" rowspan="3">
          <layout class="QGridLayout" name="gridLayout_7">
           <item row="0" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Manual File Selection* (.genbank, .fasta, .gbff, or .fna)</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>NCBI File Selection</string>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QPushButton" name="browseForFile">
             <property name="text">
              <string>Browse for a File</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="NCBI_File_Search">
             <property name="text">
              <string>Search NCBI</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item row="0" column="3" rowspan="2">
          <widget class="QPushButton" name="submitButton">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>50</height>
            </size>
           </property>
           <property name="text">
            <string>Add Job</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QGroupBox" name="Step1">
        <property name="maximumSize">
         <size>
          <width>11111</width>
          <height>218</height>
         </size>
        </property>
        <property name="title">
         <string>Step 1: Input Organism Info</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_2">
         <item row="1" column="0" rowspan="3" colspan="3">
          <layout class="QGridLayout" name="gridLayout">
           <item row="4" column="1">
            <widget class="QLineEdit" name="strainName"/>
           </item>
           <item row="4" column="0">
            <widget class="QLabel" name="Subspecies">
             <property name="text">
              <string>Subspecies/Strain</string>
             </property>
            </widget>
           </item>
           <item row="5" column="2">
            <widget class="QPushButton" name="whatsthisButton">
             <property name="text">
              <string>?</string>
             </property>
            </widget>
           </item>
           <item row="6" column="0">
            <widget class="QLabel" name="endoNameLabel">
             <property name="text">
              <string>Endonuclease*</string>
             </property>
            </widget>
           </item>
           <item row="5" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Organism Code*</string>
             </property>
            </widget>
           </item>
           <item row="6" column="1">
            <widget class="QComboBox" name="comboBoxEndo"/>
           </item>
           <item row="5" column="1">
            <widget class="QLineEdit" name="orgCode"/>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="OrganismName">
             <property name="text">
              <string>Organism Name*</string>
             </property>
            </widget>
           </item>
           <item row="4" column="3">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>5 Length*</string>
             </property>
            </widget>
           </item>
           <item row="3" column="3">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Seed Length*</string>
             </property>
            </widget>
           </item>
           <item row="4" column="4">
            <widget class="QLineEdit" name="five_length"/>
           </item>
           <item row="3" column="4">
            <widget class="QLineEdit" name="seed_length"/>
           </item>
           <item row="5" column="3">
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>3 Length</string>
             </property>
            </widget>
           </item>
           <item row="5" column="4">
            <widget class="QLineEdit" name="three_length"/>
           </item>
           <item row="6" column="3" colspan="2">
            <widget class="QCheckBox" name="repeats_box">
             <property name="text">
              <string>Generate Repeats</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="orgName"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="Line" name="line_2">
        <property name="lineWidth">
         <number>2</number>
        </property>
        <property name="midLineWidth">
         <number>1</number>
        </property>
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="label_8">
        <property name="minimumSize">
         <size>
          <width>170</width>
          <height>44</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>300</width>
          <height>44</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">font: 28pt &quot;Sans Serif&quot;;
</string>
        </property>
        <property name="text">
         <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-weight:600; color:#000000;&quot;&gt;New Genome&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0">
       <widget class="QGroupBox" name="Step3">
        <property name="title">
         <string>Step 3: Run Jobs</string>
        </property>
        <layout class="QGridLayout" name="gridLayout_8">
         <item row="0" column="0">
          <layout class="QGridLayout" name="gridLayout_5">
           <item row="6" column="2">
            <widget class="QProgressBar" name="progressBar">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="value">
              <number>0</number>
             </property>
            </widget>
           </item>
           <item row="0" column="0" colspan="2">
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QPushButton" name="remove_job">
               <property name="text">
                <string>REMOVE JOB</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="clearButton">
               <property name="text">
                <string>CLEAR ALL</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="runButton">
               <property name="text">
                <string>RUN</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Progress:</string>
             </property>
            </widget>
           </item>
           <item row="4" column="2" rowspan="2">
            <widget class="QTextBrowser" name="output_browser">
             <property name="verticalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="horizontalScrollBarPolicy">
              <enum>Qt::ScrollBarAsNeeded</enum>
             </property>
             <property name="sizeAdjustPolicy">
              <enum>QAbstractScrollArea::AdjustIgnored</enum>
             </property>
             <property name="lineWrapMode">
              <enum>QTextEdit::NoWrap</enum>
             </property>
            </widget>
           </item>
           <item row="4" column="0" rowspan="3" colspan="2">
            <widget class="QTableWidget" name="job_Table">
             <property name="minimumSize">
              <size>
               <width>500</width>
               <height>250</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>650</width>
               <height>16777215</height>
              </size>
             </property>
             <attribute name="horizontalHeaderDefaultSectionSize">
              <number>165</number>
             </attribute>
             <attribute name="horizontalHeaderMinimumSectionSize">
              <number>50</number>
             </attribute>
             <column>
              <property name="text">
               <string>Job Queue</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Job in Progress</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
             <column>
              <property name="text">
               <string>Completed Jobs</string>
              </property>
              <property name="textAlignment">
               <set>AlignCenter</set>
              </property>
             </column>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="ReqLabel">
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>15</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>15</height>
         </size>
        </property>
        <property name="text">
         <string>*Required</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="0" column="0">
     <spacer name="horizontalSpacer_3">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1151</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionExisting_Organism"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionAdd_Endonuclease"/>
   </widget>
   <widget class="QMenu" name="menuPath">
    <property name="title">
     <string>Path</string>
    </property>
    <addaction name="actionChange_Directory"/>
   </widget>
   <widget class="QMenu" name="menuStatistics">
    <property name="title">
     <string>Statistics</string>
    </property>
    <addaction name="actionView_Statistics"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="visit_repo"/>
    <addaction name="go_ncbi"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuPath"/>
   <addaction name="menuStatistics"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionExisting_Organism">
   <property name="text">
    <string>Existing Organism</string>
   </property>
  </action>
  <action name="actionAdd_Endonuclease">
   <property name="text">
    <string>Add Endonuclease</string>
   </property>
  </action>
  <action name="actionView_Statistics">
   <property name="text">
    <string>View Statistics</string>
   </property>
  </action>
  <action name="actionChange_Directory">
   <property name="text">
    <string>Change Directory</string>
   </property>
  </action>
  <action name="visit_repo">
   <property name="text">
    <string>Visit Repository</string>
   </property>
  </action>
  <action name="go_ncbi">
   <property name="text">
    <string>Go to NCBI</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
